onnxruntime/python/tools/transformers/bert_test_data.py

```diff
@@ -584,7 +584,12 @@ def create_and_save_test_data(
 
     import onnxruntime
 
-    session = onnxruntime.InferenceSession(model)
+    providers = (
+        ["CUDAExecutionProvider", "CPUExecutionProvider"]
```

Comment:
Should we include `TensorrtExecutionProvider` as an option for performance testing?

Reply:
The tool is for generating test data and save to file, not for perf testing.